dynamic_rest/serializers.py

```diff
@@ -510,7 +510,7 @@ class WithDynamicModelSerializerMixin(WithDynamicSerializerMixin):
 
     @classmethod
     def get_model(cls):
-        return cls.Meta.model
+        return getattr(cls.Meta, 'model', None)
```

Comment:
Yes, if you didn't provide `Meta` class and thus it won't have `model` attribute this will raise `AttributeError`. However, serializer may not have related model.